docker/docker-compose-e2e.yml

```diff
@@ -0,0 +1,35 @@
+# This file is only meant to be run as an override to docker-compose.yml
+# see https://docs.docker.com/compose/extends/#multiple-compose-files
+services:
+  client:
+    environment:
+      - CUTTLE_DOCKERIZED=true
+      - VITE_API_URL=http://server:1337
+      - VITE_BUILD_OUTPUT=assets
```

Comment:
If this file gets merged with the existing `docker-compose.yml` why does it need to define client here?

Reply:
It's not merged, it overrides the key, so in this case we need all the same values as far as I understand. I could be wrong though, the documentation is a little hard to parse. I can probably check it to be honest, I'll make a note to verify.